app/views/pages/home/_signin.html.haml

```diff
@@ -1,11 +1,15 @@
 %h2.govuk-heading-m= t('.title')
 
-%p= t('.description')
-
-- if ENV['FEATURE_SIGN_IN_ALERT'].to_s == 'true'
+- if !authenticated?
```

Comment:
I don't mind `if !...`, but you can use `unless` if you like

Reply:
Didn't see the `else` yet... you could switch the if and else to make it easier to read